src/styles/messages.scss

```diff
@@ -166,7 +166,7 @@
 		&.json {
 			position: relative;
 			&:before {
-				content: '{~}';
+				content: url(../../resources/icons/json.svg);
```

Comment:
json icon aligning seems to be a litte off

Reply:
I will fix it, thanks